vite.config.ts

```diff
@@ -6,38 +6,77 @@ import dts from "vite-plugin-dts";
 import svgr from "vite-plugin-svgr";
 
 export default defineConfig({
-  publicDir: "res",
   build: {
     target: browserslistToEsbuild(),
     assetsInlineLimit: 0, // To disable assets inlining as base64 in the bundle
-    lib: {
-      entry: resolve(__dirname, "src/index.ts"),
-      name: "@vector-im/compound-web",
-      fileName: "compound-web",
-    },
+
+    // We expect to be used as a dependency with a bundler, so we don't need to
+    // minify and let the bundler deal with the sourcemaps
+    sourcemap: true,
+    minify: false,
+    cssMinify: false,
+
     rollupOptions: {
-      external: ["react", "react-dom"],
+      // We are *not* running in library mode because inlines all the assets.
+      // Instead, we're running in normal mode & specifying the entrypoint here.
+
+      input: [resolve(__dirname, "./src/index.ts")],
+      // Must be in sync with the list of dependencies in the package.json
+      // It includes all dependencies except @vector-im/compound-design-tokens
+      external: [
+        "react",
+        "react-dom",
+        "react/jsx-runtime",
+        "lodash",
+        "classnames",
+        "@radix-ui/react-form",
+      ],
```

Comment:
Could we build that list dynamically by reading the `package.json` and omitting `@vector-im/compound-design-tokens` from that list.

I'm pretty certain we will fail to keep those two things in sync otherwise.